x-pack/plugins/security_solution/server/fleet_integration/handlers/validate_endpoint_package_policy.ts

```diff
@@ -22,7 +23,7 @@ export const validateEndpointPackagePolicy = (inputs: NewPackagePolicyInput[]) =
         );
       }
 
-      const maxAllowedDate = moment.utc().subtract(18, 'months');
+      const maxAllowedDate = getControlledArtifactCutoffDate();
       if (parsedDate.isBefore(maxAllowedDate)) {
         throw createManifestVersionError(
           'Global manifest version is too far in the past. Use "latest" or a date within the last 18 months. UTC time.'
```

Comment:
```suggestion
          'Global manifest version is too far in the past. Please use either "latest" or a date within the last 18 months. The earliest valid date is October 1, 2023, in UTC time.'
```
This change will require test alignment - https://github.com/elastic/kibana/blob/9d6ec1a7cb030b25732c61be19d11d8401cd235e/x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts#L452

Reply:
done https://github.com/elastic/kibana/pull/168229/commits/24856bff3fc0c67b58c8e920206e20aa946338fa